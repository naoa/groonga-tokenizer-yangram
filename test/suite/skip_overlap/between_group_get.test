register tokenizers/yangram

tokenize TokenYaBigram "つつきAきつつ" NormalizerAuto --mode GET
