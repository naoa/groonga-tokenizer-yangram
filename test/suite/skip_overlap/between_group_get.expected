register tokenizers/yangram
[[0,0.0,0.0],true]
yangram_register 2 --skip_overlap 1
[[0,0.0,0.0],"TokenYaBigramSkipOverlap"]
tokenize TokenYaBigramSkipOverlap "つつきAきつつ" NormalizerAuto --mode GET
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "つつ",
      "position": 0
    },
    {
      "value": "つき",
      "position": 1
    },
    {
      "value": "a",
      "position": 3
    },
    {
      "value": "きつ",
      "position": 4
    },
    {
      "value": "つつ",
      "position": 5
    }
  ]
]
